src/Models/Notifications/UserNotifications.php

```
@@ -85,12 +85,11 @@ public function readOne(): array
     #[Override]
     public function patch(Action $action, array $params): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$action'. (undefined)

(UnusedFormalParameter)